### PR TITLE
Updating FrameworkUDK Nuget

### PR DESCRIPTION
**Pull request by huzaifa-d**
Updated the FrameworkUDK, WinRT and Wil Nugets for the main proj

### CODE REVIEW

**Review by kythant — APPROVED** [review]

File: dev/WindowsAppSDK_DLL/WindowsAppSDK_DLL.vcxproj [hunk 1]
```diff
@@ -374,22 +374,22 @@
...
-    <Import Project="..\..\packages\Microsoft.FrameworkUdk.arm64fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.arm64fre.targets" Condition="Exists('..\..\packages\Microsoft.FrameworkUdk.arm64fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.arm64fre.targets')" />
-    <Import Project="..\..\packages\Microsoft.FrameworkUdk.x86fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.x86fre.targets" Condition="Exists('..\..\packages\Microsoft.FrameworkUdk.x86fre.10.0.22426.1000-210723-1202.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.x86fre.targets')" />
+    <Import Project="..\..\packages\Microsoft.FrameworkUdk.amd64fre.10.0.22431.1002-210816-1731.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.amd64fre.targets" Condition="Exists('..\..\packages\Microsoft.FrameworkUdk.amd64fre.10.0.22431.1002-210816-1731.ni-release-svc-reunion2110\build\native\Microsoft.FrameworkUdk.amd64fre.targets')" />
```
Comment (huichen123) [inline dev/WindowsAppSDK_DLL/WindowsAppSDK_DLL.vcxproj:377]: >Microsoft.FrameworkUdk.amd64fre [](http://example.com/codeflow?start=36&length=31)

should keep MRTCore in sync.

**Review by kythant — CHANGES_REQUESTED** [review]
Please refer to 
https://github.com/microsoft/WindowsAppSDK/pull/1132/files
on how to update it. You'll be using line 212 on vcxproj

**Review by huichen123 — APPROVED** [review]

**Review by kythant — APPROVED** [review]

**Pull request merged**